.github/workflows/test.yml

```diff
@@ -0,0 +1,47 @@
+name: test-run
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    if: ( github.event_name == 'push' ||
+        github.event.pull_request.head.repo.full_name != github.repository ) &&
+        ( github.repository == 'luc-tielen/lua-quickcheck' )
```

Comment:
Is this check needed?

Reply:
It's up to you actually. This check allows to use GH workflow for PR's and pushes in your repository only and testing will be skipped in case PR's and pushes in forks.

Reply:
Aha, good to know. I guess we can leave it like this for now.